OpenRA.Mods.Common/Traits/World/ValidateOrder.cs

```diff
@@ -29,7 +29,7 @@ public bool OrderValidation(OrderManager orderManager, World world, int clientId
 
 			if (subjectClient == null)
 			{
-				Log.Write("debug", "Order sent to {0}: resolved ClientIndex `{1}` doesn't exist", order.Subject.Owner.PlayerName, subjectClientId);
+				Log.Write("debug", "Tick {2}: Order sent to {0}: resolved ClientIndex `{1}` doesn't exist", order.Subject.Owner.PlayerName, subjectClientId, world.WorldTick);
```

Comment:
Style nit: can we please keep the formatting arguments in order? (renumber them so tick is {0})